### PR TITLE
Feat/pm 734/user friendly tabs

### PR DESCRIPTION
**Pull request by mati-1**
[PM-734](https://playmakerpro.atlassian.net/browse/PM-734)

Added a user friendly tab that can be scrolled by clicking

### CODE REVIEW

**Review by dejwi — CHANGES_REQUESTED** [review]
you almost had it😅

File: components/common/Tabs/tabs.tsx [hunk 1]
```diff
@@ -28,45 +28,70 @@ interface TabsProps extends VariantProps<typeof tabsCva> {
...
+                  })
+                } else if (firstTab === name) {
+                  e.target.scrollIntoView({
+                    block: 'start',
+                    inline: 'nearest',
+                    behavior: 'smooth',
+                  })
+                }
+              }, 100)
+            }}
```
Comment (dejwi) [inline components/common/Tabs/tabs.tsx:56]: `block` is for vertical alignment so all this doesn't actually do anything, it still scrolls to the end of text
`inline` is for horizontal alignment

solution below works like you desired
```suggestion
            onClick={(e: any) => {
              setTimeout(() => {
                e.target.scrollIntoView({
                  inline: tabs[0] === name ? 'start' : 'end',
                  behavior: 'smooth',
                })
              }, 100)
            }}
```

Comment (mati-1) [reply components/common/Tabs/tabs.tsx:56]: 😮

**Review by dejwi — APPROVED** [review]
You included some commits from overview tab branch, don't merge this until overview tab is merged

**Pull request merged**